tutorials/13_diagnostics_simulation_based_calibration.ipynb

```diff
@@ -1487,9 +1106,10 @@
    "source": [
     "In contrast to SBC and coverage based highest posterior density regions (Deistler et al., 2022),\n",
     "TARP provides a necessary *and sufficient* condition for posterior accuracy, i.e., it can also \n",
-    "detect inaccurate posterior estimators. Note, however, that it can still \"be gamed\" by just using\n",
-    "the prior as posterior estimator. Thus, we recommend using it and interpreting it like SBC, as a\n",
-    "neccessary condition. The diagnosis of posterior accuracy should always be complemented with \n",
+    "detect inaccurate posterior estimators. Note, however, that this property depends on the choice\n",
```

Comment:
Not sure why this statement "that this property depends on the choice of reference points" was added. The TARP paper makes the point, that the opposite is the case. As TARP exploits the expectation of the coverage probability, the reference point position is averaged out so to speak. The way it is used in sbi by default, we randomly draw reference points anyhow. What is the intention behind this statement then?

Reply:
Thanks for questioning this. I am referring to the overall choice of how the reference points are generated, this should be made clearer in the sentence. Quoting from the paper: 

> We have shown that this test is sufficient to prove that the inference is accurate, while other similar tests were necessary but not sufficient. We also tested the impact of the choice of ˆ p(θr|x) and the distance metric used by the TARP method and found that they do not significantly affect our results. The exception to this is the case where the posterior estimator is equal to the prior, in which case TARP only works if ˆ p(θr|x) has some dependency on x. It is left up to the user of the method to determine whether this is a risk.

The last two sentences are essential. In principle, TARP is necessary and sufficient, i.e., also tests for posterior accuracy and not only for calibration. However, if the distribution from which we generate the reference points does not depend on $x$, e.g., it is just uniformly sampled from the parameter space, then TARP can be "gamed" by the prior or an underfitted posterior estimator. Therefore, my recommendation would be treating and interpreting TARP just like SBC.

Reply:
Sounds good to me. I personally consider this a corner case which is only marginally relevant in practice. But I do agree that it should be mentioned.